RoboFile.php

```diff
@@ -595,6 +598,18 @@ public function releasePrepareGit() {
       ->run();
   }
 
+  public function releaseCreatePullRequest($version) {
+    $this->taskGitStack()
+      ->stopOnFail()
+      ->add('-A')
+      ->commit('Release ' . $version)
+      ->tag($version)
```

Comment:
Are you creating a `tag` here already or what does this do? The release is tagged automatically when created in GitHub releases. Here, I think we don't want to tag yet since the pull request may not be approved. Not sure what this does exactly though.

Reply:
Really? I always created the tag locally when releasing...  This does create a new git tag.

Reply:
The `release:publish-github` handles that. Actually, GitHub handles that - same as in UI:

![Screenshot 2019-05-28 at 11 33 21](https://user-images.githubusercontent.com/141436/58467773-8e7ba700-813c-11e9-8325-cd94f65f0816.png)